wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/implementation/DatatypeIdImpl.java

```diff
@@ -101,6 +101,11 @@ public class DatatypeIdImpl implements DatatypeIdValue {
 	 * {@link DatatypeIdValue#DT_GEO_SHAPE} in JSON.
 	 */
 	public static final String JSON_DT_GEO_SHAPE = "geo-shape";
+	/**
+	 * String used to refer to the property datatype
+	 * {@link DatatypeIdValue#DT_TIME} in JSON.
```

Comment:
It looks like this comment was copied from one (or two?) other places in the code, but this should be adapted to EDTF (this is not the property datatype).

Reply:
Ah yeah, good point. I forgot to update the copied javadoc. Sloppy...